tdvt/tdvt/tdvt.py

```diff
@@ -313,8 +313,8 @@ def enqueue_tests(ds_info, args, suite):
             test_config.run_as_perf = ds_info.run_as_perf
             test_config.tds = test_set.tds_name
             test_config.config_file = test_set.config_name
+
```

Comment:
Did you mean to add a space here?

Reply:
I did just to make it more clear. the first block is setting stuff on the conifg and the last line was adding things to the set.